nemo/collections/nlp/models/nlp_model.py

```diff
@@ -250,7 +250,12 @@ def load_from_checkpoint(
                 config_kwargs.pop('trainer')
             checkpoint[cls.CHECKPOINT_HYPER_PARAMS_KEY].update(config_kwargs)
 
-            model = cls._load_model_state(checkpoint, strict=strict, **kwargs)
+            if 'cfg' in kwargs:
+                model = cls._load_model_state(checkpoint, strict=strict, **kwargs)
```

Comment:
This check shouldn't be required but for now it's ok to keep it until we have a better solution.

Reply:
yeah, I just add this to be safe.